WalletWasabi.Tests/Helpers/MemPoolInfoGenerator.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+
+namespace WalletWasabi.Tests.Helpers
+{
+	public class MemPoolInfoGenerator
+	{
+		private static Random rnd = new Random();
```

Comment:
My IDE says that this should start with a upper-case letter.

Reply:
ok. Done.